package.json

```diff
@@ -25,36 +25,33 @@
   },
   "dependencies": {
     "@criipto/signatures": "^1.9.0",
-    "@criipto/ui-operations-status": "^1.1.1",
     "@fontsource/roboto-mono": "^4.5.8",
     "@fortawesome/fontawesome-svg-core": "^1.3.0",
     "@fortawesome/free-solid-svg-icons": "^6.0.0",
     "@fortawesome/react-fontawesome": "^0.1.17",
     "@mdx-js/mdx": "^1.6.22",
-    "@mdx-js/react": "^1.6.22",
-    "@reduxjs/toolkit": "^1.7.1",
-    "@sentry/gatsby": "^6.18.2",
+    "@mdx-js/react": "^2.0.0",
+    "@reduxjs/toolkit": "^2.5.1",
+    "@sentry/gatsby": "^7.63.0",
     "algoliasearch": "^4.12.2",
     "classnames": "^2.3.1",
     "dotenv": "^16.0.0",
-    "gatsby": "^4.11.0",
-    "gatsby-plugin-algolia": "^0.26.0",
+    "gatsby": "^5.14.1",
+    "gatsby-plugin-algolia": "^0.16.1",
     "gatsby-plugin-json-output": "^1.2.0",
-    "gatsby-plugin-manifest": "^4.11.1",
-    "gatsby-plugin-mdx": "^3.4.0",
-    "gatsby-plugin-react-helmet": "^5.4.0",
-    "gatsby-plugin-sitemap": "^5.4.0",
+    "gatsby-plugin-manifest": "^5.14.0",
+    "gatsby-plugin-mdx": "^5.14.0",
+    "gatsby-plugin-sitemap": "^6.14.0",
     "graphiql": "^1.5.16",
     "graphql-tag": "^2.12.6",
     "lodash": "^4.17.21",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-helmet": "^6.1.0",
-    "react-instantsearch-dom": "^6.22.0",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
```

Comment:
I get some warnings with incompatible versions. Might be solvable by constraining the package selector more?

Reply:
@fkj No, currently legacy-peer-deps is required, also documented by gatsby :/

Reply:
Fair enough.